elastic/apm-server
# Introduce metricset.name

## Motivation/summary

Add an optional `metricset.name` field to metrics docs. Set it to "transaction" for transaction histogram metrics, and "service_destination" for service destination (span) metrics.

## Checklist


## How to test these changes

Enable transaction and service destination aggregation, check that metric documents have a `metricset.name` field with value equal to either "transaction" or "service_destination".

## Related issues

Partially implements https://github.com/elastic/apm-server/issues/4392. Still need to handle breakdown metrics, and "app" metrics.
